jest.config.js

```diff
@@ -29,7 +29,7 @@ module.exports = {
       branches: 80,
       functions: 80,
       lines: 80,
-      statements: -15,
+      statements: -10,
```

Comment:
**Annotation:** Change Jest rules back to -10 for statements to ensure there is proper coverage.